### DISCUSSION TITLE
Mail relay for custom domain

### DISCUSSION THREAD

**Discussion by M0nk3yOo** [anonaddy/anonaddy#335] [Feature Requests]
Hi,

I've an idea for those, who have imported their custom domain to your solution:
Create ability (with auth of course) to relay mails from the custom domain.

Szenario: I've another service which needs to send mails from my custom domain. But if I moved my domain to anonaddy, I'm not able anymore to send mails from my old provider to the internet (sender unknown, of course).
Yes, I know that we've the ability to send a mail to the alias in a special format, which forwards the mail afterwards (very nice!), but you come in trouble for other services, because usually (self hosted) services are creating header from/to in a normal SMTP-format, and not replace "to" in the special formatted address.

I hope you could follow, otherwise I'll try to explain it in another way.

Best
André

**Comment by willbrowningme** [comment]
You can still send emails from your old mail provider even if your domain is with AnonAddy, you would just need to include them in your SPF record and add their DKIM record if they have one.

**Reply by M0nk3yOo** [reply]
Oh, yes, if you don't remove the domain from the provider. Didn't thought about that :-)
But for answers to the alias it is important to not have the same alias by your provider. Mailservers are usually making a recipient lookup localy and doesn't send the mail out if it found sth.

I've my mailbox by Apple (iCloud)
